main/states/busy_state.py

```diff
@@ -50,7 +50,8 @@ def on_enter(self, payload=None):
                     os.system('play ' + audio_url[6:])  # nosec #pylint-disable type: ignore
 
             if 'volume' in reply.keys():
-                subprocess.call(["amixer", "-D", "pulse", "sset", "Master", str(reply['volume'])])  # nosec #pylint-disable type: ignore
+                os.system('amixer scontrols')   # nosec #pylint-disable type: ignore
```

Comment:
Why do you drop `subprocess.call` and use `os.system`? The `subprocess.call` is [preferred](https://docs.python.org/3/library/os.html#os.system) in Python.